framework/src/play/libs/ws/HttpAsyncResponse.java

```diff
@@ -86,10 +86,8 @@ public InputStream getStream() {
     try {
       return response.getResponseBodyAsStream();
     } catch (IllegalStateException e) {
-      return new ByteArrayInputStream(new byte[] {}); // Workaround
-      // AHC's bug on
-      // empty
-      // responses
+      // This works around AHC's bug with empty responses
```

Comment:
You already touching the comment. Would you mind explaining what the acronym AHC is for? :sweat_smile:

Reply:
My best guess was AsyncHttpClient. The abbrev AHC occurs 14 times in the code base, I can expand it in all places as part of this branch.

Reply:
If you reword the comment here to include both - e.g. `AHC (AsyncHttpClient)` - then it would be OK to me. No need to expand everywhere!

Reply:
already expanded